package.json

```diff
@@ -32,12 +32,14 @@
     "url": "https://github.com/buildo/react-placeholder/issues"
   },
   "homepage": "https://github.com/buildo/react-placeholder",
-  "dependencies": {},
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "lodash.omit": "^4.5.0"
```

Comment:
Oh, right. Well, we may consider https://www.npmjs.com/package/eslint-plugin-no-unused-vars-rest, but that's probably out of scope...